lib/plugins/aws/deploy/compile/functions/index.js

```diff
@@ -124,6 +149,13 @@ class AwsCompileFunctions {
 
     _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Outputs,
       newOutputObject);
+
+    const newVersionOutput = this.cfOutputLatestVersionTemplate();
+    newVersionOutput.Value = { Ref: versionLogicalId };
+
+    _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Outputs, {
+      [`${functionLogicalId}QualifiedArn`]: newVersionOutput,
```

Comment:
I think the function logical ID makes more sense here though. If I just use function names, outputs look like this:

```
        {
            "Description": "Current Lambda function version", 
            "OutputKey": "fooQualifiedArn", 
            "OutputValue": "arn:aws:lambda:us-east-1:509803855674:function:bandlab-aws-vpc-resources-dev-foo:2"
        }, 
        {
            "Description": "Lambda function info", 
            "OutputKey": "FooLambdaFunctionArn", 
            "OutputValue": "arn:aws:lambda:us-east-1:509803855674:function:bandlab-aws-vpc-resources-dev-foo"
        },
```

IMO, this way (using logical ID) makes more sense:

```
        {
            "Description": "Lambda function info", 
            "OutputKey": "FooLambdaFunctionArn", 
            "OutputValue": "arn:aws:lambda:us-east-1:509803855674:function:bandlab-aws-vpc-resources-dev-foo"
        }, 
        {
            "Description": "Current Lambda function version", 
            "OutputKey": "FooLambdaFunctionQualifiedArn", 
            "OutputValue": "arn:aws:lambda:us-east-1:509803855674:function:bandlab-aws-vpc-resources-dev-foo:4"
        },
```